src/Domain/WsServer/Plugins/Tick/GameTick.php

```diff
@@ -103,7 +103,9 @@ private function tryTickServer(array $tickData, bool $showDebug): ?PromiseInterf
         }
 
         if (!strstr($_SERVER['REQUEST_URI'], 'dev') || Config::GetInstance()->ShouldWaitForSimulationsInDev()) {
```

Comment:
this bit can go (we don't do anything with that anymore):
!strstr($_SERVER['REQUEST_URI'], 'dev') ||

Reply:
weg